test/node/decimal128.test.ts

```diff
@@ -70,7 +72,7 @@ describe('Decimal128', function () {
   });
 
   it('fromString NaN input', function (done) {
-    var result = Decimal128.fromString('NaN');
+    let result = Decimal128.fromString('NaN');
```

Comment:
While we are here, instead of reassigning the `result` for `NaN` and `+NaN`, can we break these out into separate `it` blocks for each? So one `it` for `NaN` input and one `it` for `+NaN` input and then `result` can be a `const`. Same with the next block for `Infinity` and `+Infinity`.